adapt/engine.py

```diff
@@ -368,3 +434,41 @@ def register_intent_parser(self, intent_parser, domain=0):
             self.register_domain(domain=domain)
         self.domains[domain].register_intent_parser(
             intent_parser=intent_parser)
+
+    def drop_intent_parser(self, parser_name, domain):
+        """Drop a registered intent parser.
+
+        Arguments:
+            parser_name (str): parser name to drop.
+            domain (str): domain to drop from
+
+        Returns:
+            (bool) True if an intent parser was dropped else false.
+        """
+        return self.domains[domain].drop_intent_parser(parser_name)
+
+    def drop_entity(self, domain, **kwarg):
```

Comment:
It appears you did not define a `query` arg or kwarg for this function.

Reply:
Same as above.